src/stream.jl

```diff
@@ -47,14 +47,13 @@ end
 Base.show(io::IO, state::SheetRowStreamIteratorState) = print(io, "SheetRowStreamIteratorState( is open = $(state.is_open) , row = $(state.row) )")
 
 # Opens a file for streaming.
-@inline function open_internal_file_stream(xf::XLSXFile, filename::String) :: Tuple{ZipFile.Reader, EzXML.StreamReader}
+@inline function open_internal_file_stream(xf::XLSXFile, filename::String) :: Tuple{ZipArchives.ZipReader, EzXML.StreamReader}
     @assert internal_xml_file_exists(xf, filename) "Couldn't find $filename in $(xf.source)."
     @assert xf.source isa IO || isfile(xf.source) "Can't open internal file $filename for streaming because the XLSX file $(xf.filepath) was not found."
-    io = ZipFile.Reader(xf.source)
 
-    for f in io.files
-        if f.name == filename
-            return io, EzXML.StreamReader(f)
+    for f in ZipArchives.zip_names(xf.io)
+        if f == filename
+            return xf.io, EzXML.StreamReader(IOBuffer(ZipArchives.zip_readentry(xf.io, f, String)))
         end
     end
```

Comment:
```suggestion
    f = ZipArchives.zip_openentry(xf.io, filename)
    return xf.io, EzXML.StreamReader(f)
```
`zip_openentry` can be used here to avoid decompressing the entire entry into memory.

Also, the error on the line after this can be removed with this change.